docs/writing-stories/args.md

```diff
@@ -2,15 +2,15 @@
 title: 'Args'
 ---
 
-A story is a component with a set of arguments that define how the component is to be rendered. “Args” are Storybook’s mechanism for defining those arguments in a single JavaScript object. Args can be used to dynamically change props, slots, styles, inputs, etc. This allows Storybook and its addons to live edit components. You _do not_ need to change your underlying component code to use args.
+A story is a component with a set of arguments that define how the component should render. “Args” are Storybook’s mechanism for defining those arguments in a single JavaScript object. Args can be used to dynamically change props, slots, styles, inputs, etc. It allows Storybook and its addons to live edit components. You _do not_ need to modify your underlying component code to use args.
```

Comment:
```suggestion
A story is a component with a set of arguments that define how the component should render. “Args” are Storybook’s mechanism for defining those arguments in a single JavaScript object. Args can be used to dynamically change props, slots, styles, inputs, etc. This allows Storybook and its addons to live edit components. You _do not_ need to modify your underlying component code to use args.
```

"Args" is a strange term, grammatically. It's plural in its spelling, but singular in meaning. E.g. In "'Args' are Storybook's mechanism...", I think "is" would be technically correct, but it reads much more awkwardly than "are". Therefore, to be consistent with that decision, rather than "It allows Storybook...", we should use "This allows Storybook..." or "They allow Storybook...".